examples/Joints/Ingen/pinjoint.py

```diff
@@ -0,0 +1,16 @@
+from sympy import Symbol
+from pydy.bodies import Body
+from pydy.joints import PinJoint
+from pydy.joints_method import JointsMethod
+
+parent = Body('parent')
+child = Body('child')
+
+gravity = Symbol('gravity')
+child.add_force((0, child.get_mass() * gravity, 0), child.get_masscenter())
+
+pin_joint = PinJoint('pin_joint', parent, child, parent_point_pos=(1, 1, 1),
+                     child_point_pos=(1, 1, 1))
```

Comment:
What are you creating here? Why wouldn't you leave the parent position at (0, 0, 0) if this is a simply pendulum bob. Maybe change this to have the child point be `(0, l, 0)` or `(0, 0, 1)` (depending on which way gravity is.